### ISSUE TITLE
[BUG] Incorrect date format in template's {{date}} variable when setting Format setting

### ISSUE THREAD

**Issue by Bloodsucker** [liamcain/obsidian-periodic-notes#167]
**Issue description**
The "Format" setting in the Periodic Notes Obsidian plugin is causing the `{{date}}` variable in the configured template to use the user-specified format instead of the system default `YYYY-MM-DD`.

**Expected outcome**
The `{{date}}` variable in the configured template should use the system default format of `YYYY-MM-DD` regardless of the "Format" setting in the plugin.

**Steps to reproduce**

2. Configure the plugin to use a template with the `{{date}}` variable
3. In the plugin settings, set the "Format" setting to `YYYY/MM/YYYY-MM-DD` (or to any format you wish)
4. Create a new daily note
5. Observe that the note's path and filename are correct `YYYY/MM/YYYY-MM-DD`, however the template's `{{date}}` variable is also `YYYY/MM/YYYY-MM-DD` instead of `YYYY-MM-DD` which I believe is wrong.

**Workaround**
Don't use the core's template plugin `{{date}}` and instead use Templater plugin.

**Comment by bo-boka** [comment]
Thanks for flagging. I would greatly appreciate if this was fixed since I don't currently use the Templater plugin & am trying to keep plugins light, but I appreciate the workaround. Having the date in the note is a standard desire for a journal.

**Comment by csegeb** [comment]
Yes this is even worse when using dataview inside the note with {{date}} as a variable. Breaks the whole thing. The templater workaround is problematic because it can lead to conflicting dates (when creating a daily note for logging yesterday's activities after the fact, tp.now obviously has to be yesterday's date instead of today's). Very confusing.

In my mind, the whole folder-in-format-shenannigan is the root of the problem. After all, there IS an option to choose a folder. All it needs is the ability to input variables. The Syntax for the date should be reserved for the date!
